css-will-change-transform-rasterization/index.html

```diff
@@ -0,0 +1,58 @@
+---
+feature_name: "Rasterization &amp; will-change: transform"
+chrome_version: 53
+feature_id: 5637351992721408
+---
+
+<h3>Background</h3>
+<p>
+  All content that does not have the <code>will-change: transform</code> CSS
+  property will be re-rastered when its transform scale changes. In other words,
+  <code>will-change: transform</code> effectively means "please animate it fast"
+  without taking the additional time for rasterization. This only applies to
+  scaling that happen via JavaScript manipulation, and does not apply to CSS
+  animations.
+</p>
+
+<h3>Live Output</h3>
+<p>
+  You can see the impact in the sample text below:
+</p>
+
+{% capture html %}
+<div id="container">
+  <div id="remainsBlurry">always blurry</div>
+  <div id="noLongerBlurry">blurry before Chrome 53</div>
+  <div id="alwaysCrisp">always crisp</div>
+</div>
+{% endcapture %}
+{% include html_snippet.html html=html %}
+
+{% capture css %}
+#container {
```

Comment:
Can you remove this rule from the CSS that user sees as it doesn't add any value?
Simply add it in an inline <style> tag.